tools/chplcheck/src/chplcheck.py

```diff
@@ -53,9 +53,11 @@ def main():
         return
 
     for (filename, context) in chapel.files_with_contexts(args.filenames):
-        asts = context.parse(filename)
-        for (node, rule) in driver.run_checks(context, asts):
-            print_violation(node, rule)
+        # Silence errors, warnings etc. -- we're just linting.
```

Comment:
should this comment be here? aren't we displaying warnings/errors?

Reply:
Not with `track_errors() as errors` -- the `errors` variable will get populated with the errors / warnings, but we won't report them.